client/src/pages/Home.tsx

```diff
@@ -18,48 +18,56 @@ export function Home() {
   const [calendars, setCalendars] = useState<Calendar[]>([]);
   const [marks, setMarks] = useState<Mark[]>([]);
   const [calendarArray, setCalendarArray] = useState<JSX.Element[]>([]);
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
 
+  // creates an event listener on the window for width resizing
   useEffect(() => {
-    const localAuthJson = localStorage.getItem('um.auth');
-    if (localAuthJson) {
-      const localAuth = JSON.parse(localAuthJson);
-      handleSignIn(localAuth.user, localAuth.token);
-    }
-  }, [handleSignIn]);
+    const handleResize = () => {
```

Comment:
This is the second time I've seen this. Maybe it's time for a custom hook? Here is an article on how to do that:
https://www.dhiwise.com/post/react-get-screen-width-everything-you-need-to-know

I also wonder if you could do this with Tailwind media query modifiers. For example, would this work instead of the `windowWidth >= 700` below:
`className="hidden md:flex basis-2/5 ..."`
(See https://tailwindcss.com/docs/responsive-design if you aren't familiar with it.)